prover/lang/kore-lang.md

```diff
@@ -472,6 +472,25 @@ and values, passed to K's substitute.
   rule getLength(.Patterns) => 0
   rule getLength(P, Ps) => 1 +Int getLength(Ps)
 
+  syntax Pattern ::= getLast(Patterns) [function]
+  rule getLast(Ps) => getMember(getLength(Ps) -Int 1, Ps)
+
+  syntax Patterns ::= takeFirst(Int, Patterns) [function]
```

Comment:
`head` and `tail` may be better names?

Reply:
`head` would return only the first item from the list, while `takeFirst` returns a list of items. Similarly, `tail` returns the list of everything except head, while `getLast` returns the last item: like `head(reverse(l))`.